AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -83,56 +83,56 @@ let package = Package(
                 .awsSDKHTTPAuth,
                 .awsSDKIdentity
             ],
-            path: "./Sources/Core/AWSClientRuntime",
+            path: "Sources/Core/AWSClientRuntime/Sources",
```

Comment:
Both runtime modules and service clients have been arranged to match the internal layout of the standard Swift package.  Paths in this file have been adjusted to accommodate.

Also, service clients now have a `Resources` subfolder which is included in the target.